elastalert/util.py

```diff
@@ -390,7 +432,12 @@ def build_es_conn_config(conf):
     parsed_conf['aws_region'] = None
     parsed_conf['profile'] = None
     parsed_conf['headers'] = None
+    parsed_conf['qw_enable'] = parse_boolean_conf('qw_enable', conf)
     parsed_conf['es_host'] = os.environ.get('ES_HOST', conf['es_host'])
+
+    if parsed_conf['qw_enable']:
+        parsed_conf['es_host'] = "{}/api/v1/_elastic".format(parsed_conf['es_host'])
```

Comment:
It would be nice to move this quickwit host (URL) formatter into a common location so we don't have a maintainer accidentally forget to update each location in the code, if something changes down the road.

Reply:
Done